lib/settings/settings_screen.dart

```diff
@@ -59,6 +65,41 @@ class SettingsScreen extends StatelessWidget {
                     title: const Text(
                         'Dark text on homescreen (for bright wallpapers)'),
                   ),
+                  SettingsTile.switchTile(
+                    onToggle: (value) {
+                      value
+                          ? showColorPicker(context,
+                              onColorSelected: (Color color) {
+                              settings.setBackgroundColor(color);
+                            }, initialColor: settings.getBackgroundColor())
+                          : settings.setBackgroundColor(Colors.transparent);
+                    },
+                    initialValue:
+                        settings.getBackgroundColor() != Colors.transparent,
+                    title: const Text('Solid Wallpaper Color'),
+                  ),
+                  CustomSettingsTile(
+                    child: Visibility(
+                      visible:
+                          settings.getBackgroundColor() != Colors.transparent,
+                      child: SettingsTile(
+                        title: const Text('Selected Color'),
+                        trailing: SizedBox(
+                            height: 35,
+                            width: 35,
+                            child: Container(
+                              margin: EdgeInsets.all(3.0),
+                              decoration: BoxDecoration(
```

Comment:
add outline to add contrast against background (for case when selected color is similar to settings background color)